sample-compose/app/src/main/java/co/nimblehq/sample/compose/ui/screens/home/HomeViewModel.kt

```diff
@@ -14,25 +15,27 @@ import javax.inject.Inject
 @HiltViewModel
 class HomeViewModel @Inject constructor(
     private val useCase: UseCase,
-    dispatchers: DispatchersProvider
+    val toaster: Toaster,
```

Comment:
I don't see this variable being used in this class. 🤔

Reply:
@lydiasama Looks like it's being accessed directly from `HomeScreen` 🤔 Do you think we should make it private and have a public method for `toast()` instead? 🤔

Reply:
@ryan-conway I agree 👍🏻 . Moreover, it seems to not use the toaster here inside of this class too. So I think we can get rid of it.

Reply:
That might depend on [how we do navigation](https://github.com/nimblehq/android-templates/discussions/279) for this project tho 😉 
If we stick with fragments, then we can just inject the `Toaster` into the container fragment like usual 😄
But if we go with the new navigation method, we'll need to keep it in the `ViewModel` to stop it being too coupled with the `HomeScreen` composable 🤔

Reply:
Resolved: https://github.com/nimblehq/android-templates/discussions/279